storage/aptosdb/src/metrics.rs

```diff
@@ -59,10 +59,17 @@ pub static APTOS_STORAGE_LATEST_ACCOUNT_COUNT: Lazy<IntGauge> = Lazy::new(|| {
     .unwrap()
 });
 
-pub static APTOS_STORAGE_PRUNE_WINDOW: Lazy<IntGauge> = Lazy::new(|| {
+pub static STORAGE_PRUNE_WINDOW: Lazy<IntGauge> = Lazy::new(|| {
     register_int_gauge!("aptos_storage_prune_window", "Aptos storage prune window").unwrap()
 });
 
+pub static LEDGER_PRUNE_WINDOW: Lazy<IntGauge> =
+    Lazy::new(|| register_int_gauge!("ledger_prune_window", "Aptos ledger prune window").unwrap());
```

Comment:
I'd keep the `aptos_storage_` prefix for now, just to be consistent. -- change of prefix is a orthogonal effort.

Also, just add labels to the above counter works okay? We don't even break the dashboard too much by doing that.